src/dotless.Test/Specs/RulesetsFixture.cs

```diff
@@ -135,5 +135,135 @@ public void IESlash9HackAllowedForNonEvaluatedValues()
             var expected = input;
             AssertLess(input, expected);
         }
+
```

Comment:
Adding that logic to Output seems fine to me (perhaps by introducing an overload for AppendMany, since there seem to be quite a few of them already). And even if that turns out to be sub-optimal, that's far from the worst offender. This codebase has some gnarly bits in it. :)